digital_land/repository/entry_repository.py

```diff
@@ -188,12 +188,13 @@ def _entries_from_rows(self, entity, rows):
             rowdict = dict(row)
             resource = rowdict.pop("__RESOURCE__")
             line_num = rowdict.pop("__LINE_NUM__")
-            entry_fact[(resource, line_num)].add(
+            entry_date = rowdict.pop("__ENTRY_DATE__")
+            entry_fact[(resource, line_num, entry_date)].add(
                 Fact(entity, rowdict["attribute"], rowdict["value"])
             )
 
         result = {
-            Entry.from_facts(entity, facts, key[0], key[1])
+            Entry.from_facts(entity, facts, key[0], key[1], key[2])
```

Comment:
Could use argument expansion (not sure if it will work), i.e. 
`Entry.from_facts(entity, facts, *key)`

Reply:
nice idea - will try.